js/apps/system/_admin/aardvark/APP/react/src/views/query/queryResults/useSyncQueryExecuteJob.tsx

```diff
@@ -153,5 +152,5 @@ export const useSyncQueryExecuteJob = ({
     };
     // disabled because these functions don't need to be in deps
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [asyncJobId, index]);
```

Comment:
this fixed it because asyncJobId is stable, index keeps changing when we execute new queries